phpstan.neon

```diff
@@ -1,7 +1,5 @@
-includes:
-        - vendor/phpstan/phpstan/conf/bleedingEdge.neon
```

Comment:
Curious about the removal of this configuration?

Reply:
this was caused an error in level 4 in my local dev even the memory allocation is 2GB:

```
Reached internal errors count limit of 50, exiting...                                                                                                         
     Internal error: Child process timed out after 60.0 seconds. Try making it longer with parallel.processTimeout setting.                                        
     Child process error (exit code 255):                                                                                                                          
     Fatal error: Allowed memory size of 2147483648 bytes exhausted (tried to allocate 20480 bytes) in                                                             
     phar:///Users/samsonasik/www/CodeIgniter4/vendor/phpstan/phpstan/phpstan/vendor/nette/utils/src/Utils/Callback.php on line 58                                 
                                                                                                                                                                   
                                                                                                                                                                   
     [ErrorException]                                                                                                                                              
                                                                                                                                                                   
     Allowed memory size of 2147483648 bytes exhausted (tried to allocate 20480 bytes)                                                                             
                                                                                                                                                                   
                                                                                                                                                                   
     Fatal error: Uncaught ErrorException: Use of undefined constant FCPATH - assumed 'FCPATH' (this will throw an Error in a future version of PHP) in            
     /Users/samsonasik/www/CodeIgniter4/system/Common.php:137                                                                                                      
     Stack trace:                                                                                                                                                  
     #0 /Users/samsonasik/www/CodeIgniter4/system/Common.php(137): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', '/Users/samsonas...', 137,  
     Array)                                                                                                                                                        
     #1 /Users/samsonasik/www/CodeIgniter4/app/Views/errors/cli/error_exception.php(11): clean_path('phar:///Users/s...')                                          
     #2 /Users/samsonasik/www/CodeIgniter4/system/Debug/Exceptions.php(308): include('/Users/samsonas...')                                                         
     #3 /Users/samsonasik/www/CodeIgniter4/system/Debug/Exceptions.php(174): CodeIgniter\Debug\Exceptions->render(Object(ErrorException), 500)                     
     #4 /Users/samsonasik/www/CodeIgniter4/system/Debug/Exceptions.php(224): CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))                
     #5 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()                                                                                       
     #6 {main}                                                                                                                                                     
       thrown in /Users/samsonasik/www/CodeIgniter4/system/Common.php on line 137     
```